incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/streams/OpenapiServerFactory.java

```diff
@@ -147,28 +156,32 @@ public MessageConsumer newStream(
 
         MessageConsumer newStream = null;
 
-        if (binding != null && binding.isCompositeNamespace(NamespacedId.namespaceId(originId)))
+        if (binding != null)
         {
+            final int namespaceId = NamespacedId.namespaceId(originId);
+            if (binding.isCompositeNamespace(namespaceId))
+            {
```

Comment:
Can we remove the double if nesting and revert back to single line if?

Btw, perhaps we can move the extraction of namespaceId to implementation of binding method, so something like
```
if (binding != null && binding.isCompositeOriginId(originId))
{
...
}
```